ambari-logsearch-config-json/src/main/java/org/apache/ambari/logsearch/config/json/model/inputconfig/impl/InputS3FileDescriptorImpl.java

```diff
@@ -34,7 +34,7 @@
   @ShipperConfigElementDescription(
     path = "/input/[]/s3_access_key",
     type = "string",
-    description = "The access key used for AWS credentials."
+    description = "The access key used for AWS credentials. (Not supported yet through shipper configurations)"
```

Comment:
Is it possible to omit these not supported properties from the doc? I think these should only be included if the functionality is supported.
lines: 37, 46, 55